linkListCycle.py

```diff
@@ -0,0 +1,22 @@
+# Definition for singly-linked list.
+# class ListNode(object):
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+class Solution(object):
+    def hasCycle(self, head):
+        """
+        :type head: ListNode
+        :rtype: bool
+        """
+        if not head:
```

Comment:
Since you are checking `while fast != None` on line 17, you don't need line 13 and 14.